hemi/hemi_core.h

```diff
@@ -0,0 +1,26 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
I would consider renaming this hemi.h and renaming hemi.h hemi_core.h...  (So the convenient default is to include everything, but if you want to pare it down you can).